src/test/codegen/align-struct.rs

```diff
@@ -42,31 +42,37 @@ pub fn align64(i : i32) -> Align64 {
 // CHECK-LABEL: @align64_load
 #[no_mangle]
 pub fn align64_load(a: Align64) -> i32 {
-// CHECK: [[FIELD:%.*]] = bitcast %Align64* %{{.*}} to i32*
-// CHECK: {{%.*}} = load i32, i32* [[FIELD]], align 64
+    // CHECK: [[FIELD:%.*]] = bitcast %Align64* %{{.*}} to i32*
+    // CHECK: {{%.*}} = load i32, i32* [[FIELD]], align 64
     a.0
 }
 
 // CHECK-LABEL: @nested64
 #[no_mangle]
 pub fn nested64(a: Align64, b: i32, c: i32, d: i8) -> Nested64 {
-// CHECK: %n64 = alloca %Nested64, align 64
+    // CHECK: %_5 = alloca %Align64, align 64
+    // CHECK: %1 = bitcast %Align64* %_5 to i8*
+    // CHECK: %2 = bitcast %Align64* %a to i8*
+    // CHECK: call void @llvm.memcpy.{{.*}}(i8* align 64 %1, i8* align 64 %2, i{{[0-9]+}} 64, i1 false)
```

Comment:
This is not the right way of doing things. I think I will pass the `Nested64` to an `extern fn` to ensure it gets a place on the stack.